Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -186,19 +188,57 @@ class PlaceDetailsCardView: UIScrollView {
         if frame != newFrame { frame = newFrame }
     }
 
+    // TODO: don't duplicate PlaceCarousel code
+    func updateUI(forPlace place: Place) {
```

Comment:
In an ideal MVC world the Views should have no knowledge of Models and the Controller should set the view properties, but I am not going to be anal about it here. This is essentially fine as we are not storing the `Place` anywhere in this view.